frontend/src/components/abfragevarianten/ReportsPlanungsursaechlichkeitComponent.vue

```diff
@@ -99,6 +119,31 @@ export default class ReportsPlanungsursaechlichkeitComponent extends Mixins(Abfr
     return url.toString();
   }
 
+  private getUrlPlanungsursaechlicheSpitzenbedarfe(artBedarf: string, url: string): string {
```

Comment:
```suggestion
  private getUrlPlanungsursaechlicheSpitzenbedarfe(artBedarf: string, url: URL): string {
```

Typ vom `url`-Parameter ist `URL`.